src/components/ui/PriceDisplay.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class PriceDisplay extends React.Component {
+    static propTypes = {
+        fish: PropTypes.object.isRequired,
+        professionMultiplier: PropTypes.number.isRequired,
+    };
+
+    render() {
+        const { professionMultiplier, fish: {price} } = this.props;
+        const { base, silver, gold } = price;
+
+        return (
+            <React.Fragment>
+                <div>{Math.floor(base * professionMultiplier)}</div>
+                <div style={{color: '#a2c8ef'}}>{Math.floor(silver * professionMultiplier)}</div>
+                <div style={{color: '#f4ce50'}}>{Math.floor(gold * professionMultiplier)}</div>
+            </React.Fragment>
+        )
+    }
+}
+
+export default PriceDisplay;
```

Comment:
Configure your editor to always leave a blank line at the end of the file. This file does not have one.